cmd/config-current_test.go

```diff
@@ -174,55 +174,55 @@ func TestValidateConfig(t *testing.T) {
 		{`{"version": "` + v + `", "browser": "on", "browser": "on", "region":"us-east-1", "credential" : {"accessKey":"minio", "secretKey":"minio123"}}`, false},
 
 		// Test 11 - Test AMQP
-		{`{"version": "` + v + `", "credential": { "accessKey": "minio", "secretKey": "minio123" }, "region": "us-east-1", "browser": "on", "notify": { "amqp": { "1": { "enable": true, "url": "", "exchange": "", "routingKey": "", "exchangeType": "", "mandatory": false, "immediate": false, "durable": false, "internal": false, "noWait": false, "autoDeleted": false }}}}`, true},
```

Comment:
Does this mean that previously the tests were wrong?

Reply:
> Does this mean that previously the tests were wrong? Because , shouldPass variable has been changed from true to false in many cases

We weren't validating too much.. only now.. if you see in this case, amqp is enabled but no url is specified.

Reply:
ok